ports/mimxrt/moduos.c

```diff
@@ -35,6 +35,8 @@
 #include "genhdr/mpversion.h"
 #include "fsl_trng.h"
 
+extern const mp_obj_type_t mp_fat_vfs_type;
```

Comment:
inclued `extmod/vfs_fat.h` instead of redefining this here

Reply:
fixed.